packages/wire-format/src/__tests__/good_sample_messages.ts

```diff
@@ -27,17 +27,22 @@ export const goodMessage = {
         isFinal: false,
         outcome: [
           {
-            allocationItems: [
+            allocations: [
               {
                 amount: '0x00000000000000000000000000000000000000000000000006f05b59d3b20000',
-                destination: '0x00000000000000000000000063e3fb11830c01ac7c9c64091c14bb6cbaac9ac7'
+                destination: '0x00000000000000000000000063e3fb11830c01ac7c9c64091c14bb6cbaac9ac7',
+                allocationType: 0,
```

Comment:
Should this be the number literal `0`? It seems as though `0` has special meaning defined by `nitro-protocol`.

Reply:
fair point!